src/docs/guides/fastify.md

```diff
@@ -15,30 +15,32 @@ This guide covers how to deploy a Fastify app on Railway in four ways:
+We highly recommend that [you eject from the template after deployment](/guides/deploy#eject-from-template-repository) to create a copy of the repo on your GitHub account.
+
+**Note:** You can also choose from a <a href="https://railway.app/templates?q=fastify" target="_blank">variety of Fastify app templates</a> created by the community.
 
 ## Deploy from a GitHub Repo
 
 To deploy a Fastify app on Railway directly from GitHub, follow the steps below:
 
-1. Fork the basic <a href="https://github.com/railwayapp-templates/fastify-hello-world" target="_blank">fastify-hello-world GitHub repo</a>.
+1. Fork the basic <a href="https://github.com/railwayapp-templates/fastify" target="_blank">fastify GitHub repo</a>.
 2. Create a <a href="https://railway.app/new" target="_blank">New Project.</a>
 3. Click **Deploy from GitHub repo**.
-4. Select your `fastify-hello-world` repo.
+4. Select your `fastify` repo.
     - Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.
 5. Click **Deploy Now**.
 
-Once your Fastify app is successfully deployed, a Railway service (also referred to as an app) will be created for you. By default, this service will not be publicly accessible.
+Once your Fastify app is successfully deployed, a Railway [service](/guides/services) (also referred to as an app) will be created for you. By default, this service will not be publicly accessible.
 
-To set up a publicly accessible URL for your app, navigate to the **Networking** section in the **Settings** tab of your new service and click on **Generate Domain**.
+To set up a publicly accessible URL for your app, navigate to the **Networking** section in the [Settings](/overview/the-basics#service-settings) tab of your new service and click on [Generate Domain](/guides/public-networking#railway-provided-domain).
 
 <Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727377689/docs/languages-and-frameworks/fastifyhelloworld_xbkrry.png"
 alt="screenshot of new project menu with deploy from github selected"
 layout="responsive"
 width={2447} height={1029} quality={100} />
 
-**Note:** Railway requires that Fastify's `.listen` method for the `host` be set to `0.0.0.0` or `::`. 
-You can find this in the <a href="https://github.com/railwayapp-templates/fastify-hello-world/blob/main/index.js#L23">sample Fastify GitHub repo</a>. 
+**Note:** Railway requires that Fastify's `.listen` method for the `host` be set to `::`. This allows the app to be available over the <a href="/guides/public-networking" target="_blank">public</a> and <a href="/guides/private-networking" target="_blank">private network</a>.
+You can find this in the <a href="https://github.com/railwayapp-templates/fastify/blob/main/src/app.ts#L23" target="_blank">sample Fastify GitHub repo</a>. 
```

Comment:
The slug to a line in app.ts could become wrong over time if we ever made a change to the code that adds or removes a line.

Reply:
You're right.